### DISCUSSION TITLE
Daisy chain board with an hpdb50 connector?

### DISCUSSION THREAD

**Discussion by grichter** [akuker/RASCSI#904] [Ideas]
Anyone have a board design or a place I can get one?

I know there's adapters and cables out there, however I would like be able to have a mixed stack on my build.

**Comment by MrGasS** [comment]
Do you mean a typical 50 pin <> HPDB-25 daisy-chain board?

**Reply by grichter** [reply]
Nope. I want the higher density db50 external port typical on scsi 2 cards.

**Reply by MrGasS** [reply]
Yes, I understood the concept but, since you are talking about a daisy-chain board, I'm assuming that you would like a board like this but with an HPDB50 instead of the typical DB-25.
Am I right or I'm misunderstanding something?

![img_6804](https://user-images.githubusercontent.com/37547183/194863953-4b765233-9bc9-4124-97e2-0ba1e2595b60.jpg)

**Reply by grichter** [reply]
Yes. That's right on.

**Reply by MrGasS** [reply]
Okay, I'll try to do one during my spare time
